frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.unit.spec.tsx

```diff
@@ -74,7 +74,12 @@ const setup = ({ tag = createMockTemplateTag() }: SetupOpts = {}) => {
     { storeInitialState: state },
   );
 
-  return { setTemplateTag, setTemplateTagConfig, setParameterValue };
+  return {
+    setTemplateTag,
+    setTemplateTagConfig,
+    setParameterValue,
+    user: userEvent.setup(),
```

Comment:
was this part of the fix?
It seems [the docs](https://testing-library.com/docs/user-event/setup/#direct-apis) say that it should behave the same even if using the global object, even though they recommend using userEvent.setup

Reply:
nope, not a part of the fix, I used it for testing, but why not to keep it if it's a recommended way

Reply:
> but why not to keep it if it's a recommended way

Because this comment will not be available to anyone doing git blame in the future.

Reply:
do you mean we should revert that part?

Reply:
Ideally, yes.
Let's keep things clean and let's *please* merge this asap.